<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stationDialog</class>
 <widget class="QDialog" name="stationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="saveButtonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Call Sign</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Grid Square</string>
   </property>
  </widget>
  <widget class="QLabel" name="stationInfoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Station Information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gridSquareLineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="callSignLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>128</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="locationLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>City</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>County</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="countyLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>State</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stateLineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="countryLineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Country</string>
   </property>
  </widget>
  <widget class="QComboBox" name="serialPortComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Winkeyer Serial Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sectionLineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ARRL Section</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>callSignLineEdit</tabstop>
  <tabstop>locationLineEdit</tabstop>
  <tabstop>stateLineEdit</tabstop>
  <tabstop>gridSquareLineEdit</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>countyLineEdit</tabstop>
  <tabstop>countryLineEdit</tabstop>
  <tabstop>sectionLineEdit</tabstop>
  <tabstop>serialPortComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>saveButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>stationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>stationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
